src/worker.cc

```diff
@@ -937,12 +938,16 @@ void from_json(rapidjson::Document& doc, Options::Action action, Api& api) {
   // parse any named costings for re-costing a given path
   auto recostings = rapidjson::get_child_optional(doc, "/recostings");
   if (recostings && recostings->IsArray()) {
+    // make sure we only have unique recosting names in the end
+    std::unordered_set<std::string> names;
```

Comment:
this is incredibly nitpicky feel free to ignore:
```suggestion
    std::unordered_set<std::string> names;
    names.reserve(recostings->GetArray().Size());
```

Reply:
haha it's fine:)